Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs

```diff
@@ -114,8 +113,11 @@ protected void LoadApplication(Application application)
 				_renderersAdded = true;
 			}
 
+			if (_application != null)
+				_application.PropertyChanged -= AppOnPropertyChanged;
+
 			_application = application ?? throw new ArgumentNullException(nameof(application));
-			(application as IApplicationController)?.SetAppIndexingProvider(new AndroidAppIndexProvider(this));
+			((IApplicationController)application).SetAppIndexingProvider(new AndroidAppIndexProvider(this));
```

Comment:
Why this change? If anything, just call `SetAppIndexingProvider` directly as it is a public member of `Application` (but marked as not visible to the editor)

Reply:
My change was to remove the **?** because it was useless, then StephaneDelcroix suggered to replace the safe cast with a direct cast.